AwsCryptographicMaterialProviders/dafny/AwsCryptographicMaterialProviders/Model/cryptographic-materials-cache.smithy

```diff
@@ -151,17 +180,7 @@ structure CreateCryptographicMaterialsCacheInput {
   entryCapacity: PositiveInteger,
 
   @range(min: 0)
-  entryPruningTailSize: PositiveInteger, 
+  entryPruningTailSize: PositiveInteger,
 
-  @range(min: 0)
-  gracePeriod: PositiveInteger,
-
-  @range(min: 0)
-  graceInterval: PositiveInteger,
-
-  @range(min: 0)
-  fanOut: PositiveInteger,
-
-  @range(min: 0)
-  inFlightTTL: PositiveInteger,
+  trackerSettings : StormTrackerSettings
```

Comment:
Would it be better to express this as a union?

`StormTrackerSettings | None | Default`?
Rather than overloading `gracePeriod`?